packages/flutter/lib/src/material/input_decorator.dart

```diff
@@ -399,7 +401,6 @@ class _HelperErrorState extends State<_HelperError> with SingleTickerProviderSta
   }
 
   Widget _buildError() {
-    assert(widget.errorText != null);
```

Comment:
I think it's still useful to assert here.  If we don't then there will still be an error (on line 415 below), but this way the error is probably more clear.

```dart
assert(widget.error != null || widget.errorText != null);
```